src/components/TopBar/ExportMenu.tsx

```diff
@@ -0,0 +1,380 @@
+        alignment: 'left' as const,
+        margin: [0, 0, 0, 0],
+      } as ContentImage;
+
+      const mapLegend = {
+        image: mapLegendDataUrl,
+        width: mapWidth,
+        alignment: 'left' as const,
+        margin: [0, 0, 0, 0],
+      } as ContentImage;
+
+      const mapText = {
+        text: t('export.images.map-label'),
+        style: 'small',
+        alignment: 'left',
+        margin: [0, 6, 0, 0],
+      } as ContentText;
+
+      // Compartment table with compartment name, reference value and selected value
+      const tableBody = [
+        [
+          {text: tGlobal('export.table.compartment'), style: 'tableHeader'},
+          {text: tGlobal('export.table.reference-value'), style: 'tableHeader', alignment: 'right'},
+          {text: tGlobal('export.table.selected-value'), style: 'tableHeader', alignment: 'right'},
+        ],
+      ];
+
+      for (const compartment of compartmentNames) {
+        tableBody.push([
+          {text: compartment.name, style: 'tableCell'},
+          {
+            text: formatNumber(compartmentValues[compartment.id] ?? 0),
+            style: 'tableCell',
+            alignment: 'right',
+          },
+          {
+            text: formatNumber(cardValues[selectedScenario ?? '']?.[compartment.id] ?? 0),
+            style: 'tableCell',
+            alignment: 'right',
+          },
+        ]);
+      }
+
+      // Compartment table layout
+      const zebraLayout: TableLayout = {
+        fillColor: (rowIndex: number) => (rowIndex === 0 ? '#f5f5f5' : rowIndex % 2 === 0 ? '#fafafa' : null),
+        hLineColor: '#e0e0e0',
+        vLineColor: '#e0e0e0',
+      };
+
+      const numbersTable: ContentTable = {
+        layout: zebraLayout,
+        table: {
+          headerRows: 1,
+          widths: ['*', 'auto', 'auto'],
+          body: tableBody,
+        },
+        margin: [0, 0, 0, 4],
+      };
+
+      const mapInfoColumn: ContentColumns = {
+        alignment: 'left',
+        columns: [
+          {
+            width: '40%',
+            stack: [map, mapLegend, mapText],
+          },
+          {
+            width: '60%',
+            stack: [infoTable, numbersTable],
+          },
+        ],
+      };
+
+      docContents.push(mapInfoColumn);
+
+      // Download the pdf
+      const pdfmake = pdfMake as {createPdf?: (doc: unknown) => {download: (name: string) => void}};
+      pdfmake?.createPdf?.(doc)?.download(`ESID-export${languageSuffix}.pdf`);
+    })();
+  }, [
+    get,
+    t,
+    compartmentNames,
+    compartmentValues,
+    selectedScenarioName,
+    selectedDistrictName,
+    referenceDay,
+    cardValues,
+    selectedScenario,
+    selectedDate,
+    languageSuffix,
+    formatNumber,
+    tGlobal,
+    onDone,
+  ]);
+
+  const handleExportCsv = useCallback(() => {
+    onDone?.();
+  }, [onDone]);
+
+  return (
+    <>
+      <MenuItem onClick={handleExportPdf}>PDF</MenuItem>
+      <MenuItem onClick={handleExportCsv}>CSV (WIP)</MenuItem>
```

Comment:
I haven't put in the CSV backend yet so we should disable this until then.
```suggestion
      <MenuItem onClick={handleExportCsv} disabled>CSV</MenuItem>
```